packages/cf-style-container/README.md

```diff
@@ -2,10 +2,105 @@
 
 > Cloudflare Style Container
 
-Set of high order components for fela based applications.
+Set of high order components and other helpers for fela based applications.
 
 ## Installation
 
 ```sh
 $ npm install cf-style-container
 ```
+
+### Aliased functions from fela and react-fela
+
+We proxy/alias some useful functions from fela without changing their behaviour. See the original documentation for more details. We wrap all Fela APIs so we can eventually switch Fela to a different CSS in JS lib if ever needed.
```

Comment:
Most of them share the same camelCase CSS in JS styles, so you can use any renderer if there's better. Btw, that's why projects like polished can be shared. It's still nice to have one `cf-style-container` toolbox.